### PR TITLE
Perform PCA on SNP-chip data only

### PR DESCRIPTION
**Pull request by KatalinaBobowik**
Generate PCA of SNP chip samples only after [successfully running](https://batch.hail.populationgenomics.org.au/batches/3430/jobs/2) in the`test` bucket.

### CODE REVIEW

File: scripts/hail_batch/snp_chip_pca_sanity_check/snp_chip_only_generate_pca.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    scores_path = output_path('scores.ht')
+    loadings_path = output_path('loadings.ht')
+    # Perform PCA
+    eigenvalues, scores, loadings = hl.hwe_normalized_pca(
+        snp_chip.GT, compute_loadings=True, k=5
+    )
+    hl.Table.from_pandas(pd.DataFrame(eigenvalues)).export(eigenvalues_path)
+    scores.write(scores_path, overwrite=True)
+    loadings.write(loadings_path, overwrite=True)
+
```
Comment (lgruen) [inline scripts/hail_batch/snp_chip_pca_sanity_check/snp_chip_only_generate_pca.py:30]: Do you also want to project the TOB-WGS samples into those PCs?

**Review by lgruen — APPROVED** [review]

File: scripts/hail_batch/snp_chip_pca_sanity_check/snp_chip_only_generate_pca.py (hunk 1 above)
Comment (KatalinaBobowik) [reply scripts/hail_batch/snp_chip_pca_sanity_check/snp_chip_only_generate_pca.py:30]: Can't I just project those in the plotting script?

Comment (lgruen) [reply scripts/hail_batch/snp_chip_pca_sanity_check/snp_chip_only_generate_pca.py:30]: Sure, that works too :)

Comment (KatalinaBobowik) [reply scripts/hail_batch/snp_chip_pca_sanity_check/snp_chip_only_generate_pca.py:30]: Perfect :D

**Pull request merged**